WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -62,7 +62,7 @@ enum FeatureFlag: Int, CaseIterable, OverrideableFlag {
         case .weeklyRoundupBGProcessingTask:
             return true
         case .domains:
-            return false
+            return BuildConfiguration.current ~= [.localDeveloper, .a8cBranchTest]
```

Comment:
👋 This can now be updated to make Domains available on all Jetpack app builds in preparation for shipping in Jetpack 19.9.